src/library/Uno.Material/Styles/Controls/ComboBox.xaml

```diff
@@ -0,0 +1,429 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+					xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:xamarin="http://nventive.com/xamarin"
+					xmlns:toolkit="using:Uno.UI.Toolkit"
+					mc:Ignorable="xamarin">
+
+	<!-- System Resources (Comment/Uncomment/Add/Remove resources you need)-->
+	<SolidColorBrush x:Key="ComboBoxBackgroundColorBrush"
+					 Color="{ThemeResource SurfaceColor}" />
+
+	<SolidColorBrush x:Key="ComboBoxPointerOverBackgroundThemeBrush"
+					 Color="{StaticResource OnSurfaceColor}"
+					 Opacity="0.04" />
+	<SolidColorBrush x:Key="ComboBoxSelectedBackgroundThemeBrush"
+					 Color="{StaticResource OnSurfaceColor}"
+					 Opacity="0.04" />
+	<SolidColorBrush x:Key="ComboBoxSelectedPointerOverBackgroundThemeBrush"
+					 Color="{StaticResource OnSurfaceColor}"
+					 Opacity="0.04" />
+	<SolidColorBrush x:Key="ComboBoxPressedHighlightThemeBrush"
+					 Color="{StaticResource OnSurfaceColor}"
+					 Opacity="0.04" />
+	<SolidColorBrush x:Key="ComboBoxPressedBackgroundThemeBrush"
+					 Color="{StaticResource OnSurfaceColor}"
+					 Opacity="0.04" />
+
+	<SolidColorBrush x:Key="ComboBoxBorderBrush"
+					 Color="{StaticResource OnSurfaceColor}"
+					 Opacity="0.12" />
+
+	<SolidColorBrush x:Key="ComboBoxArrowForegroundThemeBrush"
+					 Color="{ThemeResource OnSecondaryColor}"
+					 Opacity="0.6" />
+	<SolidColorBrush x:Key="ComboBoxForegroundThemeBrush"
+					 Color="{ThemeResource OnSecondaryColor}" />
+	<SolidColorBrush x:Key="ComboBoxFocusedBackgroundThemeBrush"
+					 Color="{ThemeResource OnSecondaryColor}"
+					 Opacity="0.6" />
+
+	<SolidColorBrush x:Key="ComboBoxPlaceholderTextForegroundThemeBrush"
+					 Color="{ThemeResource OnSecondaryColor}"
+					 Opacity="0.6" />
```

Comment:
all those colors should point to the proper brush directly, not recode the opacity.

Like this` <StaticResource x:Key="TextBoxLabelDefaultColorBrush" ResourceKey="OnSurfaceMediumBrush" />`